packages/web/components/table/assets-table.tsx

```diff
@@ -0,0 +1,253 @@
+              }
+            },
+      };
+    },
+    [sortKey, sortDirection, toggleSortDirection, setSortKey, setSortDirection]
+  );
+
+  // User toggles for showing 10+ pools and assets with > 0 fiat value
+  const [showAllPools, setShowAllPools] = useState(false);
+  const [hideZeroBalances, setHideZeroBalances] = useState(false);
+
+  // Filter data based on user's input in the search box.
+  const [query, setQuery, filteredSortedCells] = useFilteredData(
+    hideZeroBalances
+      ? sortedCells.filter((cell) => cell.amount !== "0")
+      : sortedCells,
+    ["chainName", "chainId", "coinDenom", "amount", "fiatValue", "queryTags"]
+  );
+
+  const tableData = showAllPools
+    ? filteredSortedCells
+    : filteredSortedCells.slice(0, 10);
+
+  return (
+    <section className="bg-surface min-h-screen">
+      <div className="max-w-container mx-auto p-10">
+        <div className="flex place-content-between">
+          <h5>Osmosis Assets</h5>
+          <div className="flex gap-5">
+            <SearchBox
+              currentValue={query}
+              onInput={(query) => {
+                setHideZeroBalances(false);
+                setQuery(query);
+              }}
+              placeholder="Filter by symbol"
+            />
+            <SortMenu
+              selectedOptionId={sortKey}
+              onSelect={setSortKey}
+              options={[
+                {
+                  id: "coinDenom",
+                  display: "Symbol",
+                },
+                {
+                  /** These ids correspond to keys in `Cell` type and are later used for sorting. */
+                  id: "chainName",
+                  display: "Network",
+                },
+                {
+                  id: "amount",
+                  display: "Balance",
+                },
+              ]}
+            />
+          </div>
+        </div>
+        <Table<TableCell>
+          className="w-full my-5"
+          columnDefs={[
+            {
+              display: "Asset / Chain",
+              displayCell: AssetNameCell,
+              sort: sortColumnWithKeys(["coinDenom", "chainName"]),
+            },
+            {
+              display: "Balance",
+              displayCell: BalanceCell,
+              sort: sortColumnWithKeys(["amount", "fiatValue"], "descending"),
+              className: "text-right pr-24",
+            },
+            {
+              display: "Deposit",
+              displayCell: (cell) => (
+                <TransferButtonCell type="deposit" {...cell} />
+              ),
+              className: "text-center max-w-[5rem]",
+            },
+            {
+              display: "Withdraw",
+              displayCell: (cell) => (
+                <TransferButtonCell type="withdraw" {...cell} />
+              ),
+              className: "text-center max-w-[5rem]",
+            },
+          ]}
+          rowDefs={tableData.map(() => ({ makeHoverClass: () => " " }))}
```

Comment:
It would be better to remove hover styling to make the user aware nothing happened if clicking table row.